tests/dummy/app/controllers/nodes/detail/files/provider/file.js

```diff
@@ -5,6 +5,21 @@ export default Ember.Controller.extend({
         reloadFiles() {
             this.transitionToRoute('nodes.detail.files.provider',
                 this.get('node'), this.model.get('provider'));
+        },
+        addATag(tag) {
+            var file = this.get('model');
+            var currentTags = file.get('tags').slice(0);
```

Comment:
Ok. @hmoco's latest work may fix this. I'll add a TODO to check into this when his changes are ready.